products/veeam-backupandreplication

```diff
@@ -0,0 +1,111 @@
+---
+title: Veeam Backup & Replication
+category: app
+tags: veeam
+iconSlug: veeam
+permalink: /veeam-backupandreplication
+alternate_urls:
+-   /veeambackupandreplication
+releasePolicyLink: https://www.veeam.com/product-lifecycle.html
+activeSupportColumn: End of Fix
+releaseColumn: false
+releaseDateColumn: true
+eolColumn: End of support
+
+releases:
+-   releaseCycle: "12.0"
+    releaseDate: 2023-02-01
+    support: true
+    eol: 2026-02-01
+    link: "https://www.veeam.com/kb4420"
+
+-   releaseCycle: "11.0"
+    releaseDate: 2021-02-01
+    support: 2023-02-01
+    eol: 2024-02-01
+    link: null
+
+-   releaseCycle: "10.0"
+    releaseDate: 2020-02-01
+    support: 2021-02-01
+    eol: 2023-02-01
+    link: null
+
+-   releaseCycle: "9.5"
+    releaseDate: 2016-01-01
+    support: false
+    eol: true
+    link: null
+
+-   releaseCycle: "9.0"
+    releaseDate: 2016-01-01
+    support: false
+    eol: true
+    link: null
+
+-   releaseCycle: "8.0"
+    releaseDate: 2014-01-01
+    support: false
+    eol: true
+    link: null
+
+-   releaseCycle: "7.0"
+    releaseDate: 2013-01-01
+    support: false    
+    eol: true
+    link: null
+
+-   releaseCycle: "6.5"
+    releaseDate: 2012-01-01
+    support: false
+    eol: true
+    link: null
+
+-   releaseCycle: "6.0"
+    releaseDate: 2011-01-01
```

Comment:
Should be 2011-08-22 according to https://www.veeam.com/kb2680.

Other release dates are wrong too (6.5 for example), could you re-check versions against https://www.veeam.com/kb2680 ?